src/tests/system/tests/test_ldap.py

```diff
@@ -14,179 +14,10 @@
 from sssd_test_framework.topology import KnownTopology
```

Comment:
Two tests introduced by this commit are missing:

1. test_ldap__ppolicy_user_login_then_changes_password
2. test_ldap__ppolicy_user_login_then_changes_password_complexity_requirement

Reply:
If those tests have anything to do with 1980e2c415666650cdb57b73055603bfd4549474 (I don't know) then that might be intentional.

Reply:
Let's see what @danlavu has to say about this. But i find strange that a commit that was cherry picked (as marked in the commit's comment) there are so many things that don't match the original commit.

Reply:
Amended cherry-pick should have 'Amended-by' like with a description what (and why) was changed.

Reply:
It is intentional, ppolicy is not supported in this version, and now that the test is covered by the generic provider there is no need to have the tests at all, so I amended the commit and removed the tests. When I ran ```git cherry-pick -x $commit_id``` I corrected all the conflicts. I will add the amended by line.

FWIW, the difference between the master 2-9 is quite different. I've attached the difference between the branches.   

[sssd-2-9-master-diff.txt](https://github.com/user-attachments/files/19762454/sssd-2-9-master-diff.txt)